### PR TITLE
[BT-1] - First Sample PR

### PR DESCRIPTION
**Pull request by murad-boratech**
Anything

### CODE REVIEW

**Comment by LenaChen7** [comment]
sup

**Comment by javi-lopezz** [comment]
dont approve pr

**Comment by MarvLopez** [comment]
'_'

**Comment by Tsai-Anthony** [comment]
cool

**Comment by ITgirlLL** [comment]
GREAT

**Comment by Turhunjohn** [comment]
awesome! what are u doing?

**Review by LenaChen7 — APPROVED** [review]
LG

**Review by MoeMo01 — APPROVED** [review]
LGTM !

**Review by KrystalOm — APPROVED** [review]
good job!

**Review by Xiaoligithub168 — APPROVED** [review]
LGTM!

**Review by HuiYang37 — APPROVED** [review]
LGTM

**Review by Minh999999 — APPROVED** [review]
LGTM!

**Review by Tsai-Anthony — APPROVED** [review]
Looks good to me @murad-boratech

**Review by Nishankeyum — APPROVED** [review]
Expected better

File: README.md
Comment (MarvLopez) [inline README.md:1]: its good

**Review by Oussema1312 — APPROVED** [review]
LGTM! KO

**Review by javi-lopezz — APPROVED** [review]
LetsGetTheMoney 
LGTM

**Review by Karma336 — APPROVED** [review]
awesome....

**Review by helen-ahn — APPROVED** [review]
LGTM

**Review by aliceoh324 — APPROVED** [review]
LGTM!!

**Comment by Turhunjohn** [comment]
LGTM!

**Review by jinyjoh — COMMENTED** [review]
great job

**Review by MarvLopez — APPROVED** [review]

**Review by ardal-akbar — APPROVED** [review]
LGTM!

**Review by Tsai-Anthony — CHANGES_REQUESTED** [review]
Need changes

**Review by LenaChen7 — CHANGES_REQUESTED** [review]
format

**Comment by Nanfeise** [comment]
nice

**Review by LenaChen7 — APPROVED** [review]

**Review by Tsai-Anthony — APPROVED** [review]

**Comment by Turhunjohn** [comment]
Please merge my branch! LGTM!

**Pull request merged**